mimecast/plugin.spec.yaml

```
@@ -369,6 +375,27 @@ actions:
         description: Managed URL that was created
         type: '[]managed_url'
         required: false
+  delete_managed_url:
+    title: Delete Managed URL
+    description: Delete a managed URL
+    input:
+      url:
+        title: URL
+        description: The URL to unblock or permit. This must match a URL that is already in
+        type: string
+        required: true
+      delete_all_entries:
+        title: Delete all matching block URLs
+        description: Remove all blocked URLs that match the input.  Otherwise, only 0 or 1 entries are deleted
```

0 entries sounds weird, I think what we might mean to say is only a single matched entry is deleted.
```suggestion
        description: True to remove all instances of the matched URL e.g. when there are duplicates. If set to false, only a single matched entry will be deleted
```

---

and then regenerate